nomad/structs/structs.go

```diff
@@ -4797,6 +4797,12 @@ func (j *Job) Warnings() error {
 		mErr.Errors = append(mErr.Errors, err)
 	}
 
+	// cron -> crons
+	if j.Periodic.Spec != "" {
```

Comment:
```suggestion
	if j.Periodic != nil && j.Periodic.Spec != "" {
```

Ooops, one more change needed 😄